src/domains/dashboard/components/PortfolioBreakdown/PortfolioBreakdown.tsx

```diff
@@ -110,7 +110,7 @@ export const PortfolioBreakdown: React.VFC<PortfolioBreakdownProperties> = ({
 	return (
 		<>
 			<div
-				className="-mx-8 flex flex-col bg-theme-secondary-100 px-8 py-4 dark:bg-black sm:mx-0 sm:rounded-xl sm:px-4 lg:flex-row"
+				className="-mx-8 flex flex-col bg-theme-secondary-100 px-6 py-4 dark:bg-black sm:mx-0 sm:rounded-xl lg:flex-row"
```

Comment:
```suggestion
				className="-mx-6 flex flex-col bg-theme-secondary-100 px-6 py-4 dark:bg-black sm:mx-0 sm:rounded-xl lg:flex-row"
```

the portfolio element is not aligned with the navbar on mobile, seems to be due to the negative margin that's used (-32px):
<img width="455" alt="image" src="https://github.com/user-attachments/assets/70e1f9df-ed0b-4289-b61c-180ab485854d">


changing it to -1.5rem/-24px seems to fit, not sure if that has other implications?:
<img width="411" alt="image" src="https://github.com/user-attachments/assets/62ffe19c-0114-4213-86f2-46e5febbd82e">

Reply:
Fixed it. Thanks